description/launch/circuit.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
since we have `iarrc_worlds.launch`, we don't need `circuit.launch`, `drag_race.launch` or `urban.launch` (please remove them)